frontend/src/component/playground/Playground/AdvancedPlayground.tsx

```diff
@@ -24,7 +24,7 @@ import { AdvancedPlaygroundResponseSchema } from 'openapi';
 export const AdvancedPlayground: VFC<{}> = () => {
     const { environments: availableEnvironments } = useEnvironments();
     const theme = useTheme();
-    const matches = useMediaQuery(theme.breakpoints.down('lg'));
+    const matches = true;
```

Comment:
UI breaks on limited space we have in the right area so forcing mobile view for now

Reply:
With `breaks`, do you mean the overflow ?